### PR TITLE
Crud implementation for books

### PR DESCRIPTION
**Pull request by oksana-feshchenko**

### CODE REVIEW

File: books/admin.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
 
-# Register your models here.
+from books.models import Book
+
+admin.site.register(Book)
```
Comment (danylo-d) [inline books/admin.py]: add blank line

File: books/views.py [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+from rest_framework import mixins, viewsets
+from books.models import Book
+from books.serializers import BookSerializer
+
+
+class BookViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    mixins.RetrieveModelMixin,
+    viewsets.GenericViewSet,
```
Comment (danylo-d) [inline books/views.py]: In our task, we should realize full CRUD 
image

**Review by danylo-d — CHANGES_REQUESTED** [review]

**Review by danylo-d — APPROVED** [review]
GJ

**Review by WDemidenko — CHANGES_REQUESTED** [review]
Where is books.models file?

**Pull request closed without merging**